ng2-components/ng2-activiti-form/src/components/activiti-form.component.ts

```diff
@@ -396,12 +388,10 @@ export class ActivitiForm implements OnInit, AfterViewChecked, OnChanges {
                 .saveTaskForm(this.form.taskId, this.form.values)
                 .subscribe(
                     () => {
-                        this.formSaved.emit(this.form);
+                        this.onTaskSaved(this.form);
```

Comment:
No, it's not a breaking change. the new method raises multiple events in a single place.